src/Backend/Form/Type/MetaType.php

```diff
@@ -52,19 +53,37 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add(
                 'title',
                 TextType::class,
-                ['label' => 'lbl.PageTitle', 'label_attr' => ['class' => 'sr-only']]
+                [
+                    'label' => 'lbl.PageTitle',
+                    'label_attr' => ['class' => 'sr-only'],
+                    'constraints' => [
+                        new NotBlank(),
```

Comment:
why do you place the constraints here and not on the data? either the entity or DTO, same for all the other ones